test/export/test_export_nonstrict.py

```diff
@@ -0,0 +1,51 @@
+# Owner(s): ["module: dynamo"]
+
+from torch.export import export
+
+try:
+    from . import test_export, testing
```

Comment:
Why do we have try/catch here?

Reply:
I copied the coding style from https://github.com/pytorch/pytorch/blob/4471fe6c3915f4afea918e1b3af95e7298ad42b4/test/dynamo/test_dynamic_shapes.py#L10-L31. Do you know why? Maybe it's related to being imported by other files. 

I think we can remove it for our purpose here.

Reply:
Yeah let's remove it. Seems weird